apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -74,6 +76,7 @@ type ClusterSpec struct {
 	DefaultClusterRoleForProjectMembers  string                               `json:"defaultClusterRoleForProjectMembers,omitempty" norman:"type=reference[roleTemplate]"`
 	DockerRootDir                        string                               `json:"dockerRootDir,omitempty" norman:"default=/var/lib/docker"`
 	EnableNetworkPolicy                  *bool                                `json:"enableNetworkPolicy" norman:"default=false"`
+	EnableClusterMonitoring              *bool                                `json:"enableClusterMonitoring" norman:"default=false"`
```

Comment:
why is this a pointer also

Reply:
similar to `EnableNetworkPolicy`, keep them in the same style.